tests/integration_tests/network/test_net_config_load.py

```diff
@@ -0,0 +1,28 @@
+"""Test loading the network config"""
+import pytest
+
+from tests.integration_tests.instances import IntegrationInstance
+
+
+def _customize_envionment(client: IntegrationInstance):
+    # Insert our "disable_network_config" file here
+    client.write_to_file(
+        "/etc/cloud/cloud.cfg.d/99-disable-network-config.cfg",
+        "network: {config: disabled}\n",
+    )
+    client.execute("cloud-init clean --logs")
+    client.restart()
+
+
+def test_network_disabled_via_etc_cloud(client: IntegrationInstance):
+    """Test that network can be disabled via config file in /etc/cloud"""
+    if client.settings.CLOUD_INIT_SOURCE == "IN_PLACE":
+        pytest.skip(
+            "IN_PLACE not supported as we mount /etc/cloud contents into the "
+            "container"
+        )
+    _customize_envionment(client)
+
+    # assert client.execute("test -f /etc/netplan/50-cloud-init.yaml").failed
```

Comment:
Did you still want this assert here if so I suspect you'd also need a remove of /etc/netplan/50-cloud-init.yaml up in _customize_environment below cloud-init clean --logs

Reply:
Whoops, no. I meant to remove this. I initially tried that and then discovered that on a "normal" instance, removing the netplan config is a bad idea :)